README.md

```diff
@@ -46,6 +60,77 @@ apps:
       openAIKey: $OPENAI_API_KEY
 ```
 
+
+### Using Azure OpenAI
+
+To provision the plugin to use Azure OpenAI, use settings similar to this:
+
+```yaml
+apiVersion: 1
+
+apps:
+  - type: 'grafana-llm-app'
+    disabled: false
+    jsonData:
+      openAI:
+        provider: azure
+        url: https://<resource>.openai.azure.com
+        azureModelMapping:
+          - ["gpt-3.5-turbo", "gpt-35-turbo"]
+    secureJsonData:
+      openAIKey: $OPENAI_API_KEY
+```
+
+where:
+
+- `<resource>` is your Azure OpenAI resource name
+- the `azureModelMapping` field contains `[model, deployment]` pairs so that features know
+  which Azure deployment to use in place of each model you wish to be used.
+
+### Provisioning vector services
+
+The vector services of the plugin allow certain AI-based features to perform semantic search to improve the responses given by LLMs. Configuration is in roughly three parts:
```

Comment:
We may want to skip the vector service in docs until the pieces are in place

Reply:
TBH I think the pieces _are_ in place, but if you'd like we can do two separate releases? Feel free to remove this section if so